app/views/albums/show.html.erb

```diff
@@ -5,14 +5,15 @@
 <%# .variant %>
 <p>
   <div style='width: 500px; height:auto; margin:0 auto'>
+  <%= image_tag(@album.image, alt: 'User Image', style: 'width:100%') %>
```

Comment:
the image through Imaage didn't work, so instead I've attached the image to the Albums